cfgov/v1/jinja2/v1/enforcement-action/index.html

```diff
@@ -26,10 +26,6 @@
     {%- endfor %}
 {% endblock %}
 
-{% block content_sidebar_modifiers -%}
-    o-sidebar-content enforcement-action-sidebar
```

Comment:
FYI I didn't find any reference to `enforcement-action-sidebar`, other than its creation here.